common/system.h

```diff
@@ -1295,6 +1298,21 @@ class OSystem : Common::NonCopyable {
 	 */
 	virtual Common::String getSystemLanguage() const;
 
+	/**
+	 * Suggest textures to render at the side of the game window.
+	 * This enables eg. Grim to render the game in a widescreen format.
+	 * 
+	 * The system must take ownership or the pointers will be freed
+	 * automatically.
+	 *
+	 * !!! ResidualVM specific method: !!!
+	 *
+	 * @param left			Texture to be used on the left
+	 * @param height		Texture to be used on the right
+	 */
+	virtual void suggestSideTextures(Common::ScopedPtr<Graphics::Surface>& left,
```

Comment:
I'm a bit unsure about passing Common::ScopedPtr references as parameters. It works fine, but feels a bit strange to me.